test/views/SubmitFormViewSpec.scala

```diff
@@ -158,7 +158,7 @@ class SubmitFormViewSpec extends ViewBaseSpec {
         Some("ABC Studios"),
         flatRateScheme = false,
         obligation,
-        SubmitVatReturnForm.nineBoxForm,
+        SubmitVatReturnForm().nineBoxForm,
```

Comment:
Line 178 of this file also needs these brackets adding - its a new test that's been added in a PR so you'll need to rebase